docs/installation/README.md

```diff
@@ -0,0 +1,5 @@
+# Installing Loki
+
+1. [Installing using Ksonnet (recommended)](./ksonnet.md)
+2. [Installing through Helm](./helm.md)
+3. [Installing locally](./local.md)
```

Comment:
This order of subsections is different than in README.md, which is:

	    1. [Installing Locally](installation/local.md)
	    2. [Installing with Helm](installation/helm.md)
	    3. [Installing with Ksonnet](installation/ksonnet.md)